src/main/resources/conf/fdpConfig.yml

```diff
@@ -20,10 +20,8 @@ metadataProperties:
 fairSearch:
     fdpSubmitUrl: http://localhost:8080/fse/submitFdp
 threadPoolSize: 4
-metadataFM:
-    F1:
-        A: http://example.com/THIS_IS_A_PH_URI_4_ID_SCHEME
-        B: http://example.com/THIS_IS_A_PH_URI_4_ID_PERSISTENCE
-    A1:
-        A: http://example.com/THIS_IS_A_PH_URI_4_ACCESS_PROTOCOL
-    A2: http://example.com/THIS_IS_A_PH_URI_4_MD_LONGEVITY
+metadataMetrics:
+    https://raw.githubusercontent.com/FAIRMetrics/Metrics/master/FM_F1A: https://www.ietf.org/rfc/rfc3986.txt
+    https://raw.githubusercontent.com/FAIRMetrics/Metrics/master/FM_F1B: http://example.com/THIS_IS_A_PH_URI_4_ID_PERSISTENCE
+    https://raw.githubusercontent.com/FAIRMetrics/Metrics/master/FM_A1.1: https://www.wikidata.org/wiki/Q8777
+    https://raw.githubusercontent.com/FAIRMetrics/Metrics/master/FM_A2: http://example.com/THIS_IS_A_PH_URI_4_MD_LONGEVITY
```

Comment:
We could also use the purl url as the key? (`https://purl.org/fair-metrics/` namespace)